vNext/shared/AppInsightsCommon/src/Interfaces/IConfig.ts

```diff
@@ -149,6 +150,15 @@ export interface IConfig {
      */
     disableCorrelationHeaders?: boolean;
 
+    /**
+     * @description Sets the distributed tracing modes. If AI_AND_W3C mode or W3C mode is set, W3C trace context headers (traceparent/tracestate) will be parsed in all outgoing requests. 
+     * The existing back-compatibility AI headers will also be parsed and included. Enabling W3C mode will not break existing correlation with other Application Insights instrumented services. Default=AI
```

Comment:
Delete `parsed`. We are not doing any parsing here

Reply:
Enabling W3C only mode will break existing correlation with non-w3c compatible App Insights instrumented services. If a user is worried about this happening, we should point them to use the AI+W3C setting.